collectors/health_test.go

```diff
@@ -615,7 +615,27 @@ $ sudo ceph -s
 			{
 				"count": 10,
 				"state_name": "scrubbing+deep"
-			}
+			},
+            {
```

Comment:
could you add a few other odd combinations here for testing:
active+undersized+remapped+backfill_wait
active+undersized+remapped+backfilling
active+recovery_wait+degraded+remapped+inconsistent
active+recovering+undersized+degraded+remapped

Reply:
Since it's just `strings.Contains()` not worried too much about combinations or order. :)